### PR TITLE
removed unnecery GameCollection require and double of User require th…

### PR DESCRIPTION
**Pull request by KorolyovaLara**
…at was crashing the server

### CODE REVIEW

**Comment by tinkubansal95** [comment]
Yes, we can remove these two, User is duplicate and we aren't using GameCollection any more,

**Review by tinku-bansal — APPROVED** [review]
Yes, we can remove these two, User is duplicate and we aren't using GameCollection any more,

File: routes/api/gameApi.js [hunk 1]
```diff
@@ -3,10 +3,8 @@ const router = require("express").Router();
 const UserGame = require("../../models/UserGame");
 const Games = require("../../models/Games");
 const Users = require("../../models/Users");
-const GameCollection = require("../../models/GameCollection");
```
Comment (strahinjaajvaz) [inline routes/api/gameApi.js:6]: Didn't we remove these in the other PR?

Comment (KorolyovaLara) [reply routes/api/gameApi.js:6]: Screen Shot 2021-06-08 at 11 26 17 am
Thought so, but when i pulled main, it was there and and i just followed the errors till it run again

**Pull request merged**